noxfile.py

```diff
@@ -44,6 +44,9 @@
     "eland/ml/_optional.py",
     "eland/ml/_model_serializer.py",
     "eland/ml/ml_model.py",
+    "eland/ml/pytorch/__ini__.py",
```

Comment:
Typo in `__init__`

Reply:
I just type so fast and the `t` fell off.